app/main/views/performance.py

```diff
@@ -29,10 +30,15 @@ def performance():
         ],
         key=itemgetter("organisation_name"),
     )
+    stats.pop("services_using_notify")
     stats["average_percentage_under_10_seconds"] = mean(
         [row["percentage_under_10_seconds"] for row in stats["processing_time"]] or [0]
     )
     stats["count_of_live_services_and_organisations"] = status_api_client.get_count_of_live_services_and_organisations()
+
+    if request.endpoint == "main.json":
```

Comment:
should this not be:
```suggestion
    if request.endpoint == "main.performance_json":
```

(nb: i haven't run this locally to test)

Reply:
Fixed now and added a rudimentary test that would have caught this